src/rust/engine/Cargo.toml

```diff
@@ -3,9 +3,14 @@ version = "0.0.1"
 name = "engine"
 authors = [ "Pants Build <pantsbuild@gmail.com>" ]
 
+[profile.release]
+debug = true
```

Comment:
Meant to be permanent. It doesn't cause an appreciable change in the binary size (a few hundred bytes), but makes profiling and debugging possible. And it doesn't affect whether optimization is enabled.